bin/node/runtime/src/lib.rs

```diff
@@ -223,6 +223,8 @@ impl frame_system::Config for Runtime {
 	type OnSetCode = ();
 }
 
+impl pallet_randomness_collective_flip::Config for Runtime {}
```

Comment:
Why do we need this?

Reply:
randomness-collective-flip [was refactored](https://github.com/paritytech/substrate/pull/9061) and accroding to the comment we need to add `impl pallet_randomness_collective_flip::Config for Runtime {}`